internal/authflow/flow.go

```diff
@@ -109,7 +109,7 @@ type cfg struct {
 	token string
 }
 
-func (c cfg) Token(hostname string) (string, string) {
+func (c cfg) ActiveToken(hostname string) (string, string) {
```

Comment:
I take it this is intended to be more meaningful as its possible even more so now that a user has multiple accounts with tokens, which was true being logged into GHES and GHEC.

Reply:
Well, previously there could only be one `Token` per hostname. Now it's intended to be meaningful because there is only one `active` one at a time (the one that is used for any API operations or as provided by the credential helper).